go.mod

```diff
@@ -5,7 +5,7 @@ go 1.22.5
 require (
 	cloud.google.com/go/kms v1.18.3
 	github.com/AlecAivazis/survey/v2 v2.3.7
-	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.13.0
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.12.0
```

Comment:
bad merged? this seems to be a rollback of the update

Reply:
Good catch, updated